src/main/java/nl/hsac/fitnesse/slim/AllArgScenarioTable.java

```diff
@@ -0,0 +1,59 @@
+package nl.hsac.fitnesse.slim;
+
+import fitnesse.testsystems.TestExecutionException;
+import fitnesse.testsystems.slim.HtmlTable;
+import fitnesse.testsystems.slim.SlimTestContext;
+import fitnesse.testsystems.slim.Table;
+import fitnesse.testsystems.slim.tables.SlimAssertion;
+import fitnesse.testsystems.slim.tables.SlimTable;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Scenario table that allows usage @{*} symbol to expand all input variables as a constructor parameter for
+ * fixtures. See ArgTableTemplateTest1.wiki for example usage.
+ */
+public class AllArgScenarioTable extends AutoArgScenarioTable {
+    private static final String ALL_ARG_PATTERN = "@{*}";
+
+    public AllArgScenarioTable(Table table, String tableId, SlimTestContext testContext) {
+        super(table, tableId, testContext);
+    }
+
+    @Override
+    public List<SlimAssertion> call(Map<String, String> scenarioArguments,
+                                    SlimTable parentTable, int row) throws TestExecutionException {
+        /* pass all input arguments to super inputs to bypass "Variables not used in scenario" error */
+        for(Map.Entry<String,String> entry : scenarioArguments.entrySet()){
+            if(!super.getInputs().contains(entry.getKey())){
```

Comment:
I believe the explicit `super.` prefix is not needed here (nor in the line below)

Reply:
Removed prefix